docs/content/concepts/layer/implementation/ssp/_index.md

```diff
@@ -66,6 +66,46 @@ The figure below expresses represents the portion of the OSCAL stack as it relat
 
 {{<partialCached "note-to-developers-uuid.html" >}}
 
+### Identifier References
+
+An OSCAL SSP may contain references to information that is defined locally (e.g., in the SSP model) or externally (e.g., in a referenced profile, catalog, or component definition model).  The following lists the mechanisms through which objects in an OSCAL SSP may "link" to other OSCAL content:
+
+- **Local**: Scenario where an identifier references information in the same OSCAL instance.  This scenario supports  “compile time” enforcement of constraints since the validity of the reference can be checked as soon as the SSP is authored.
```

Comment:
```suggestion
- **Local**: Scenario where an identifier references information in the same OSCAL instance.  This scenario supports  enforcement of constraints since the validity of the reference can be checked as soon as the SSP is authored.
```